lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -1,4 +1,5 @@
 import { FullScreenSpinner, LabeledButton } from '@hypothesis/frontend-shared';
+import { apiCall } from '../utils/api';
```

Comment:
Can you move this to above the `../utils/format` import below. Imports are grouped into third party, then local. Within a group, imports are sorted by path.